doc/btp_csis.txt

```diff
@@ -0,0 +1,59 @@
+CSIS Service (ID 17)
+===================
+
+Commands and responses:
+
+	Opcode 0x00 - Error response
+
+	Opcode 0x01 - Read Supported Commands command/response
+
+		Controller Index:	<controller id>
+		Command parameters:	<none>
+		Response parameters:	<supported commands> (variable)
+
+		Each bit in response is a flag indicating if command with
+		opcode matching bit number is supported. Bit set to 1 means
+		that command is supported. Bit 0 is reserved and shall always
+		be set to 0. If specific bit is not present in response (less
+		than required bytes received) it shall be assumed that command
+		is not supported.
+
+		In case of an error, the error response will be returned.
+
+	Opcode 0x02: CSIS Set Member Lock
+
+		Controller Index:	<controller id>
+		Command parameters:	Address_Type (1 octet)
+					Address (6 octets)
+					Lock (1 octet)
+					Force (1 octet)
+		Response parameters:	<None>
+
+		Address Type is a byte; Address type of the PTS address.
+		Address is a 6 octet address; Address of the PTS.
+		Lock is a boolean; True to set the Lock; False to release the Lock.
+		Force is a boolean; True to force the Lock operation; False otherwise.
+		Force is only relevant when releasing the member Lock.
+		In this case it will force release the lock, regardless of who took the lock.
+
+		This command is used to set the member Lock.
+
+		In case of an error, the error response will be returned.
+
+	Opcode 0x03: CSIS Get Member RSI
+
+		Controller Index:	<controller id>
+		Command parameters:	Address_Type (1 octet)
+					Address (6 octets)
+		Response parameters:	Status (4 octets)
```

Comment:
This is a bit unorthodox for BTP since for errors we have dedicated error response, so your return RSI on success and BTP error response on error

Reply:
No - It will always return a response with Status and RSI.  It will never return a BTP error response.
As stated below - "In case of an error, the Status field in the response will reflect the error."

Reply:
generic BTP code in zephyr and mynewt will return error eg. if size mismatch.  I'd prefer if we keep BTP commands behavior consistent regarding error handling.

Is this status some mesh specific status (4 bytes)?

Reply:
No - this is the normal int error code returned by bt_csip_set_member_generate_rsi().

Reply:
Changed the CSIS Get Member RSI function to comply to the normal error response functionality.